magenta/models/onsets_frames_transcription/model.py

```diff
@@ -133,7 +133,7 @@ def model_fn(features, labels, mode, params, config):
   length = features.length
   spec = features.spec
 
-  is_training = mode == tf.estimator.ModeKeys.TRAIN
+  is_training = (mode == tf.estimator.ModeKeys.TRAIN or mode == tf.estimator.ModeKeys.EVAL)
```

Comment:
`is_training` should be true only for the TRAIN mode. It affects things like whether dropout is enabled.

Reply:
but with evaluate mode, parameters are fixed.

Reply:
Right, but dropout affects activations, so it will still cause problems if it's turned on during evaluation.